templates/status/status-messages.html.twig

```diff
@@ -25,6 +25,7 @@
     'description': _self.message_list(messages),
     'variant': (type == 'status' ? 'success' : type),
     'extra_attributes': extra_attributes,
+    'extra_classes': 'ecl-u-mv-l',
```

Comment:
I assume this is to ofset the class we removed on page.html, but why was that done?

Reply:
removed